### PR TITLE
List ZBit (ZBT)

### PR DESCRIPTION
**Pull request by MidoriDaria**
- Official project URL: https://zbitofficial.io/
- Official block explorer URL: http://explorer.zbitofficial.io/

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/ZBit.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.AltCoinAccountDisclaimer;
+import bisq.asset.Coin;
+import bisq.asset.I18n;
+import bisq.asset.RegexAddressValidator;
+
+@AltCoinAccountDisclaimer("account.altcoin.popup.ZBT.msg")
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/ZBit.java:25]: Missing English translation of this message.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Review by blabno — APPROVED** [review]
ACK

**Review by ripcurlx — DISMISSED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/3560#pullrequestreview-319183359

**Comment by ripcurlx** [comment]
@MidoriDaria You need to sign your commit https://help.github.com/articles/signing-commits-using-gpg before I'm able to merge it.

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - because of https://github.com/bisq-network/bisq/pull/3560#issuecomment-556977035

**Pull request closed without merging**